docs/source/using/troubleshooting.adoc

```diff
@@ -113,6 +113,15 @@ cause passwords to fail.
 Workaround: When creating and entering passwords, wrap the string with
 single quotes in the following format: `<password>`
 
+NOTE: The characters `", $, ` and \` are still interpreted by the shell,
```

Comment:
> We have the password string, right? All we need is a single function which takes the password and escapes it to ssh standards.

@hferentschik That make sense and I think mostly we just need to convert that string to a ssh-acceptable so end user doesn't have issue. might be simple regex or replace works, have to check.